elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/AsyncIT.java

```diff
@@ -219,98 +214,134 @@ public void jsonApiRequestTests() throws InterruptedException {
+         .then()
+         .statusCode(org.apache.http.HttpStatus.SC_OK);
+
+        AsyncDelayStoreTransaction.sleep = false;
         int i = 0;
         while (i < 1000) {
             Thread.sleep(10);
-            Response response = given()
-                    .accept("application/vnd.api+json")
-                    .get("/asyncQuery/ba31ca4e-ed8f-4be0-a0f3-12088fa9263c");
-
-            // If Async Query is created and completed then validate results
-            if (response.jsonPath().getString("data.attributes.status").equals("COMPLETE")) {
-
-                // Validate AsyncQuery Response
-                response
-                        .then()
-                        .statusCode(HttpStatus.SC_OK)
-                        .body("data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263c"))
-                        .body("data.type", equalTo("asyncQuery"))
-                        .body("data.attributes.queryType", equalTo("GRAPHQL_V1_0"))
-                        .body("data.attributes.status", equalTo("COMPLETE"))
-                        .body("data.relationships.result.data.type", equalTo("asyncQueryResult"))
-                        .body("data.relationships.result.data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263c"));
-
-                // Validate AsyncQueryResult Response
-                given()
-                        .accept("application/vnd.api+json")
-                        .get("/asyncQuery/ba31ca4e-ed8f-4be0-a0f3-12088fa9263c/result")
-                        .then()
-                        .statusCode(HttpStatus.SC_OK)
-                        .body("data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263c"))
-                        .body("data.type", equalTo("asyncQueryResult"))
-                        .body("data.attributes.contentLength", notNullValue())
-                        .body("data.attributes.responseBody", equalTo("{\"data\":{\"book\":{\"edges\":"
-                                + "[{\"node\":{\"id\":\"1\",\"title\":\"Ender's Game\"}},"
-                                + "{\"node\":{\"id\":\"2\",\"title\":\"Song of Ice and Fire\"}},"
-                                + "{\"node\":{\"id\":\"3\",\"title\":\"For Whom the Bell Tolls\"}}]}}}"))
-                        .body("data.attributes.status", equalTo(200))
-                        .body("data.relationships.query.data.type", equalTo("asyncQuery"))
-                        .body("data.relationships.query.data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263c"));
-
+            String graphQLStatus = document(
+                    selection(
+                            field(
+                                    "asyncQuery",
+                                    arguments(
+                                            argument("ids", "\"edc4a871-dff2-4054-804e-d80075cf828e\"")
+                                    ),
+                                    selections(
+                                            field("status")
```

Comment:
Let's consolidate this request with the next request (there is no need to separate out the status call from the other bits).